astropy/modeling/functional_models.py

```diff
@@ -610,18 +610,12 @@ class Sersic1D(Fittable1DModel):
     amplitude = Parameter(default=1)
     r_eff = Parameter(default=1)
     n = Parameter(default=4)
-    _gammaincinv = None
 
-    def __init__(self, amplitude=amplitude.default, r_eff=r_eff.default,
-                 n=n.default, **kwargs):
-        try:
-            from scipy.special import gammaincinv
-            self.__class__._gammaincinv = gammaincinv
-        except ValueError:
-            raise ImportError("Sersic1D model requires scipy > 0.11.")
-
-        super(Sersic1D, self).__init__(
-            amplitude=amplitude, r_eff=r_eff, n=n, **kwargs)
+    try:
+        from scipy.special import gammaincinv
+        _gammaincinv = gammaincinv
+    except ValueError:
```

Comment:
This should probably also catch an `ImportError`. The tests are failing because `scipy` was not installed. The `ValueError` here and elsewhere is to catch the case of `python3` and `scipy < 0.12`. However, I'll add some other general comments.